drivers/mssql/mssql.go

```diff
@@ -414,7 +454,7 @@ ORDER BY i.index_id
 		}
 		r.Table = table
 		for _, c := range l.columns {
-			column, err := table.FindColumnByName(c)
+			column, err := table.FindColumnByName(strings.TrimSpace(c))
```

Comment:
I think it would be better to trim at https://github.com/k1LoW/tbls/pull/310/files#diff-1c6a9273e8cd55f36cf9e47485d049deeb0e1a634afc8a9ebd3970b842ce2bf6R296 or [Query](https://github.com/k1LoW/tbls/pull/310/files#diff-1c6a9273e8cd55f36cf9e47485d049deeb0e1a634afc8a9ebd3970b842ce2bf6R232-R264) than here.

Reply:
It appears to me that my code changes may not be doing the primary key correctly.  Would you agree based on the test results in lines 807-812?

```
| Name | Type | Definition |
 | ---- | ---- | ---------- |
-| PK__users_* | PRIMARY KEY | CLUSTERED, unique, part of a PRIMARY KEY constraint, [ id ] |
-| UQ__users_* | UNIQUE | NONCLUSTERED, unique, part of a UNIQUE constraint, [ email ] |
-| UQ__users_* | UNIQUE | NONCLUSTERED, unique, part of a UNIQUE constraint, [ username ] |
+| PK__users_* | PRIMARY KEY | CLUSTERED, unique, part of a PRIMARY KEY constraint, [  id, username, email ] |
+| UQ__users_* | UNIQUE | NONCLUSTERED, unique, part of a UNIQUE constraint, [  id, username, email ] |
+| UQ__users_* | UNIQUE | NONCLUSTERED, unique, part of a UNIQUE constraint, [  id, username, email ] |
 | CK__users__username_* | CHECK | CHECK(len([username])>(4)) |
```

Reply:
> It appears to me that my code changes may not be doing the primary key correctly.

It certainly seems that neither the primary key field nor the unique constraint field is being retrieved correctly.
( mssql DDL is [here](https://github.com/k1LoW/tbls/blob/master/testdata/ddl/mssql.sql) )

There seem to be two problems, 

1. [spaces](https://github.com/k1LoW/tbls/pull/310#pullrequestreview-660299312) 
2. incorrect field retrieval.